readme.md

```diff
@@ -26,6 +26,25 @@ const terminalImage = require('terminal-image');
 })();
 ```
 
+Optionally, you can specify the `height` and/or `width` to scale the image.
+
+```js
+const terminalImage = require('terminal-image');
+
+(async () => {
+	console.log(await terminalImage.file('unicorn.jpg', {width: 1024, height: 1024}));
```

Comment:
These example sizes are unrealistically large. The width/height are columns/rows, so use a smaller number.